app/views/posts/_checkbox_form.html.erb

```diff
@@ -0,0 +1,19 @@
+<%= form_for [@community, @post] do |form| %>
```

Comment:
Can you rename this file to something more relevant to the feature?
Name it as _content_warning_fields.html.erb or something similar @SrinandhiniM